src/audio/copier/copier.c

```diff
@@ -930,9 +940,18 @@ static int copier_reset(struct comp_dev *dev)
 	cd->output_total_data_processed = 0;
 
 	for (i = 0; i < cd->endpoint_num; i++) {
-		ret = cd->endpoint[i]->drv->ops.reset(cd->endpoint[i]);
+		if (dev->ipc_config.type == SOF_COMP_HOST) {
+			if (cd->hd->chan)
+				notifier_unregister(cd->endpoint[i],
+						    cd->hd->chan, NOTIFIER_ID_DMA_COPY);
+
+			host_zephyr_reset(cd->hd, cd->endpoint[i]->state);
+			cd->endpoint[i]->state = COMP_STATE_READY;
+		} else {
+			ret = cd->endpoint[i]->drv->ops.reset(cd->endpoint[i]);
```

Comment:
ditto for comment and wrapper.

Reply:
Please share more information for API wrapper.